test/Sentry.Tests/SentryClientTests.cs

```diff
@@ -289,6 +290,62 @@ public void CaptureEvent_DisposedClient_ThrowsObjectDisposedException()
             _ = Assert.Throws<ObjectDisposedException>(() => sut.CaptureEvent(null));
         }
 
+        [Fact]
+        public void CaptureUserFeedback_Ignored_If_EventIdEmpty()
```

Comment:
The convention is: `MethodName_StateUnderTest_ExpectedOutcome`:

```suggestion
        public void CaptureUserFeedback_EventIdEmpty_FeedbackIgnored()
```

Reply:
ty